meta-efi-secure-boot/recipes-core/images/kernel-initramfs-efi-secure-boot.inc

```diff
@@ -8,30 +8,52 @@ fakeroot python do_sign() {
     if d.getVar('BUNDLE', True) == '0':
         for compr in d.getVar('INITRAMFS_FSTYPES').split():
             uks_sel_sign(d.expand('${D}/boot/${INITRAMFS_IMAGE}${INITRAMFS_EXT_NAME}.') + compr, d)
+            uks_boot_sign(d.expand('${D}/boot/${INITRAMFS_IMAGE}${INITRAMFS_EXT_NAME}.') + compr, d)
     else:
         uks_sel_sign(d.expand('${D}/boot/${KERNEL_IMAGETYPE}-initramfs${INITRAMFS_EXT_NAME}'), d)
+        uks_boot_sign(d.expand('${D}/boot/${KERNEL_IMAGETYPE}-initramfs${INITRAMFS_EXT_NAME}'), d)
```

Comment:
My previous comment about one signing function to handle both applies here (and I think this starts to show why we need a change).

Reply:
I further had to add a new variable called ${SB_FILE_EXT} to deal with all the coping code due to the signature files having different names.   

I also made it a bb.fatal() if UEFI_SELOADER and GRUB_SIGN_VERIFY are set to one.  You only get to choose one boot loader signing tool for the final stage before the kernel is loaded.